MDANSE_GUI/Src/MDANSE_GUI/MolecularViewer/MolecularViewer.py

```
@@ -531,9 +557,91 @@ def clear_trajectory(self, clear_isosurfaces=True):
 
         del self._actors
 
+    def create_atom_label_actors(self):
+        """Creates atom label actors, setting the text to the chosen
+        atom_label_type.
+        """
+        self.atom_label_actors = []
+        if self._reader is None:
+            return
```

Silently returning may not be the best thing here, do we want to raise or warn on invalid/conflicting input?

---

Maybe, I think we should leave this for until we decide to refactor the MolecularViewer. Its possible for the reader to be None if there is no trajectory loaded in the 3D viewer. I return none here because the an attempt to create atom labels occurs when the atom label option is changed.